src/common/low_precision_transformations/tests/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
 endif()
 if(NOT MSVC)
     ie_add_compiler_flags(-Wno-missing-declarations)
-    ie_add_compiler_flags(-Wno-sign-compare)
+    ie_add_compiler_flags(-Wsign-compare)
```

Comment:
We need to add this option here https://github.com/openvinotoolkit/openvino/blob/master/cmake/developer_package/compile_flags/os_flags.cmake#L316-L363 and fix all warnings